UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -441,25 +441,24 @@ private void Add_Field_OnClick(object sender, RoutedEventArgs e)
                     // Taking the targetClass string and finding a corresponding class in diagram.
                     Class currentClass = _activeDiagram.GetClassByName(targetClass);
                     
-                    // If class does not exist...
-                    if (currentClass is null)
+                    // If input is empty...
+                    if (targetClass is null || targetClass.Trim().Length == 0)
                     {
                         RaiseAlert(
-                            "Field Creation Failed",
+                            "Field Creation Failed", 
                             "Could Not Create Field",
-                            "class does not exist",
+                            "The target class cannot be empty",
                             AlertIcon.ERROR
                         );
                         return;
                     }
-                    
-                    // If input is empty...
-                    if (targetClass is null || targetClass.Trim().Length == 0)
+                    // If class does not exist...
+                    if (currentClass is null)
                     {
                         RaiseAlert(
-                            "Field Creation Failed", 
+                            "Field Creation Failed",
                             "Could Not Create Field",
-                            "The target class cannot be empty",
+                            $"class '{targetClass}' does not exist",
                             AlertIcon.ERROR
                         );
                         return;
```

Comment:
(Referring to the block of code below at line 469)

Why do we need checks for "multiple arguments" provided for a class name (or for that matter a field name or type)?There is no way to explicitly provide multiple arguments for any of those- there is only one input box for those things. 

The only situation where this would happen is when the user puts spaces in the provided name. I don't think an error message of "Only one argument expected for target class" is helpful to the user in this situation. This is an issue of the name being provided being invalid- the check for if the name contains spaces should happen in the name validation code and not within the handlers here. We are just cluttering our handlers with if statements. I also don't think we need the "isCorrectNumArguments" method, but if we keep it the "i" in "is" should be capitalized.